app/models/bookmark.rb

```diff
@@ -35,6 +100,7 @@ def self.auto_delete_preferences
     if: Proc.new { |b| b.will_save_change_to_post_id? || b.will_save_change_to_for_topic? }
 
   validate :polymorphic_columns_present, on: [:create, :update]
+  validate :polymorphic_type_ok, on: [:create, :update]
```

Comment:
```suggestion
  validate :valid_bookmarkable_type, on: [:create, :update]
```
I feel like `ok` sounds very vague and doesn't reveal the intend of the method well.

Reply:
Yup fair enough!